src/pipecat/services/tavus.py

```diff
@@ -0,0 +1,140 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+
+"""This module implements Tavus as a sink transport layer"""
+
+import aiohttp
+import base64
+
+from pipecat.frames.frames import (
+    Frame,
+    TTSAudioRawFrame,
+    TransportMessageUrgentFrame,
+    TTSStartedFrame,
+    TTSStoppedFrame,
+    StartInterruptionFrame,
+)
+from pipecat.processors.frame_processor import FrameDirection, FrameProcessor
+
+from loguru import logger
+
+
+class TavusVideoService(FrameProcessor):
```

Comment:
We should probably inherit from `AIService` instead of `FrameProcessor` directly.